.github/workflows/tycho.yml

```diff
@@ -101,5 +105,12 @@ jobs:
         uses: actions/setup-python@v6
         with:
           python-version-file: "./src/tycho/.python-version"
+      - name: Wait for PostgreSQL
+        run: |
+          until pg_isready -h localhost -p 5432 -U postgres; do
+            echo "Waiting for PostgreSQL..."
+            sleep 2
+          done
+          echo "PostgreSQL is ready!"
```

Comment:
That's odd that you need to do this as the service health checks should do it upfront for you.